File: Submissions/#0590_div3/A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
 
  int q;
  cin >> q;

  vector<int> sum(q);
  for (int i = 0; i < q; i++) {
    int n;
    cin >> n;
    for (int j = 0; j < n; j++) {
      int a;
      cin >> a;
      sum[i] += a;
    }
    cout << (int) ceil((double) sum[i] / n) << endl;
  }

}

File: Submissions/#0590_div3/A/main.java
import java.util.*;

public class Main {

  public static void main (String[] args) {

    Scanner sc = new Scanner(System.in);
    int q = sc.nextInt();

    int[] sum = new int[q];
    for (int i = 0; i < q; i++) {
      int n = sc.nextInt();
      for (int j = 0; j < n; j++) {
        sum[i] += sc.nextInt();
      }
      System.out.println((int) Math.ceil((double) sum[i] / n));
    }
    
    sc.close();
  }

}